src/sql/parts/jobManagement/views/jobsView.component.ts

```diff
@@ -129,20 +129,8 @@ export class JobsViewComponent implements AfterContentChecked {
 		};
 
 		this.dataView = new Slick.Data.DataView({ inlineFilters: false });
-		let rowDetail = new RowDetailView({
-			cssClass: 'detailView-toggle',
-			preTemplate: this.loadingTemplate,
-			process: (job) => {
-				(<any>rowDetail).onAsyncResponse.notify({
-					'itemDetail': job,
-				}, undefined, null);
-			},
-			panelRows: 2,
-			postTemplate: () => ''
-		});
-
-		this.rowDetail = rowDetail;
 
+		this.rowDetail = new RowDetailView({});
```

Comment:
why don't we need the RowDetailView configuration parameters?  How are we showing the error messages on the Jobs List page?

Reply:
I'm gonna send another PR for the implementation of errors. This is to get rid of the bug for now.